integration_tests/mockApis/responses/incomingMovemnts.ts

```diff
@@ -59,4 +59,14 @@ export default [
     fromLocation: 'Leeds',
     moveType: 'PRISON_REMAND',
   },
+  {
+    firstName: 'Steve',
+    lastName: 'Smith',
+    dateOfBirth: '1985-05-05',
+    prisonNumber: 'G0015GF',
+    pncNumber: '01/5568B',
+    date: '2021-09-01',
+    fromLocation: 'Manchester',
+    moveType: 'COURT_APPEARANCE',
+  },
```

Comment:
Added this extra movement to be able to check the service still handled 'OTHER' incoming movements as the PRISON_REMAND moveType is now being grouped as FROM_ANOTHER_ESTABLISHMENT.